README.md

```diff
@@ -27,6 +24,31 @@ Global Level: toggle the preference of vscode, and add `gitlink.defaultRemote: "
 
 ## Release Notes
 
+### 0.5.0
+
+* Add support for azure DevOps urls
+
+### 0.4.6
```

Comment:
I made quick summaries for older versions, not sure how accurate it is. I figure it's good to list all versions of the plugin because it makes it appear more active.